### PR TITLE
Example paragraph code

### PR DESCRIPTION
**Pull request by SuryaPanneer04**
example

### CODE REVIEW

File: demo.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+<body>
+
+<p>This is a paragraph.</p>
```
Comment (iam-sundar) [inline demo.html:5]: Please remove duplicate lines.

Comment (SuryaPanneer04) [reply demo.html:5]: this is for example

Comment (SuryaPanneer04) [reply demo.html:5]: this is for example

Comment (iam-sundar) [reply demo.html:5]: Even though it's a example code. Kindly remove duplicate lines of code.

Comment (SuryaPanneer04) [reply demo.html:5]: removing duplicate lines

**Pull request merged**